data/persian/bbc_persian_radio/liveradio.json

```
@@ -45,7 +50,7 @@
   "promo": {
     "name": "رادیو فارسی بی‌بی‌سی",
     "summary": "مجله خبری بخش فارسی رادیو بی‌بی‌سی را هر روز از ساعت ۶ تا ۸ صبح به وقت تهران (۲:۳۰ تا ۴:۳۰ به وقت گرینیچ) بشنوید. این برنامه شامل تازه‌ترین خبرهای روز ایران و جهان، به همراه گزارش، گفت وگو و تحلیل و تفسیر درباره رویدادهای ایران، منطقه و جهان است.",
-    "uri": "/persian/bbc_persian_radio/liveradio",
+    "uri": "/pashto/bbc_pashto_radio/liveradio",
```

Is this meant to be pashto?

---

Don't think so - this is the persian fixture.  Pashto is the file above

Fixed the spaces - thanks for noticing those :)

---

it is `Persian` 👍

---

```suggestion
    "uri": "/persian/bbc_persian_radio/liveradio",
```

This also needs to be addressed in the Ares JSON data

---

Sorry misunderstood the point - thought you were suggesting we change this to pashto, not that it was already incorrectly set to pashto

Fixed this and sent request to ARES to fix it too